Bottom: fbc52d2dc6808279c05eeb6c58c5bc566a843627
Top:    194417d3d2ec3f8bcecd3e9c7dbbfe8e4ede3642
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 18:44:36 +0800

Refresh of autofs-5.0.7-try-and-cleanup-after-dumpmaps.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 595d93e..6849272 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2136,6 +2136,8 @@ int main(int argc, char *argv[])
 	}
 
 	if (dumpmaps) {
+		struct master_mapent *entry;
+		struct list_head *head, *p;
 		struct mapent_cache *nc;
 
 		open_log();
